packages/algob/test/builtin-tasks/compile.ts

```diff
@@ -95,8 +100,22 @@ describe("Compile task", () => {
   it("should recompile all files when --force is used", async () => {
     await op.resetAndCompile(true);
 
-    assert.equal(op.timestamp, 5);
-    assert.lengthOf(op.compiledFiles, 2);
-    assert.lengthOf(op.writtenFiles, 2);
+    assert.equal(op.timestamp, 9);
+    assert.lengthOf(op.compiledFiles, 4);
+    assert.lengthOf(op.writtenFiles, 4);
+  });
+
+  it("should return TEAL code", async () => {
+    const pyOp = new PyCompileOp(op);
+    const content = fs.readFileSync(path.join(ASSETS_DIR, f4), 'utf8');
+    const res = pyOp.compilePyTeal(f3PY) + '\n';
+    assert.deepEqual(content.toString(), res);
+  });
+
+  it("should ", async () => {
+    const pyOp = new PyCompileOp(op);
+    const result: PyASCCache = await pyOp.ensureCompiled(f3PY, false);
+    const expected = fs.readFileSync(path.join(ASSETS_DIR, 'gold-asa-py-check.yaml'), 'utf8');
+    assert.deepEqual(YAML.stringify(result), expected);
```

Comment:
add a check that `op.compile` was called. You have examples how to do it in previous tests.

Reply:
adding

Reply:
Done, added `true` in force and updated timestamp to check that `op.compiled` was called